scripts/gen_help_html.lua

```diff
@@ -977,6 +978,10 @@ local function gen_css(fname)
       margin-right: 0;
       float: right;
     }
+    .help-tag a,
+    .help-tag-right a {
+      color: inherit;
```

Comment:
text-decoration: none;

Reply:
Wouldn't it be nice to have some other visual indication that it's clickable then, besides cursor turning into pointer on hover? That's the only reason I didn't add that.

Reply:
I think that makes actual links less meaningful. There are many of these too. And most of them go "nowhere" (except the current page). So it's very noisy to decorate them as clickable things.

People who want to get the anchor will naturally hover over the heading to try that.

Reply:
Makes sense. Added that as well as a TODO comment

Reply:
Didn't see a comment edit initially. Changed rule to only remove underline if link is not hovered.

Reply:
I don't think it's needed for `:focus` though, because browsers have a separate outline when navigating with a keyboard.